src/main/java/seedu/address/logic/parser/AddMedConCommandParser.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NRIC;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MEDCON;
+
+import seedu.address.logic.commands.AddMedConCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.Nric;
+import seedu.address.model.person.MedCon;
+
+/**
+ * Parses user input for the {@link AddMedConCommand} and creates a new instance of it.
+ */
+public class AddMedConCommandParser implements Parser<AddMedConCommand> {
+    /**
+     * Parses the given arguments string and creates a {@link AddMedConCommand} object.
+     *
+     * @param args the arguments string containing user input.
+     * @return A {@link AddMedConCommand} object containing the parsed NRIC and medical conditon.
+     * @throws ParseException if the user input does not conform to the expected format or
+     *         if the NRIC or medical condition is not provided.
+     */
+    public AddMedConCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+        String trimmedArgs = args.trim();
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args,
+                PREFIX_NRIC, PREFIX_MEDCON);
+        System.out.println(trimmedArgs);
```

Comment:
Use a debugger over print statements as you might forget to remove these print statements. Please remove it in the next commit.

Reply:
Yes, I second this 👍🏻